src/lib/variants.test.ts

```diff
@@ -80,196 +80,212 @@ const targetingDefault: EpicTargeting = {
     mvtId: 2,
 };
 
+describe('find variant', () => {
+    it('should find the correct variant for test and targeting data', () => {
+        const tests = [testDefault];
+        const targeting: EpicTargeting = {
+            ...targetingDefault,
+            weeklyArticleHistory: [{ week: 18330, count: 45 }],
+        };
+
+        const got = findTestAndVariant(tests, targeting);
+
+        expect(got?.test.name).toBe('example-1');
+        expect(got?.variant.name).toBe('control-example-1');
+    });
+
+    it('should return undefined when no matching test variant', () => {
+        const test = { ...testDefault, excludedSections: ['news'] };
+        const tests = [test];
+        const targeting = { ...targetingDefault, sectionName: 'news' };
```

Comment:
Really minor, but I'm a fan of the [arrange, act, assert](https://github.com/testdouble/contributing-tests/wiki/Arrange-Act-Assert) pattern for tests. With each step separated by whitespace (like in the first test here). I reckon an extra newline here would enhance readability.

Reply:
Cool, thanks @tjmw. I thought it would read slightly better if the act and assert blocks were close together (but separated from the arrange section) but if there's an established pattern around how to separate these by white spaces, I don't mind adopting it.